pkg/client/client.go

```diff
@@ -50,6 +52,15 @@ type FetchUpdate struct {
 	ResourceCount uint64
 	// Error if any returned by the provider
 	Error string
+	// PartialFetchResults contains the partial fetch results for this update
+	PartialFetchResults []*cqproto.PartialFetchFailedResource
+}
+
+// FetchFinishCallbackRequest represents a request for the FetchFinishCallback
+type FetchFinishCallbackRequest struct {
```

Comment:
nit: this isn't the request but the result no? FetchSummary perhaps?

Reply:
I thought this is more the Request for the Callback function.